ViAn/GUI/mainwindow.cpp

```diff
@@ -212,9 +214,7 @@ MainWindow::~MainWindow() {
     delete project_wgt;
     delete analysis_wgt;
     delete bookmark_wgt;
-    delete status_bar;
```

Comment:
Is it better  to delete object manually like this or let the program clean them up by itself? They are still deleted as the code is now.